### PR TITLE
feat: lw13d4

### PR DESCRIPTION
**Pull request by w3b6x9**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Supabase Queues

### CODE REVIEW

File: apps/www/data/products/modules/queues.tsx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+      'Create Queues and manage messages in the Dashboard as well as monitor your Queues and message processing in real-time.',
+    cta: {
+      label: 'Start message queuing',
+      url: 'https://supabase.com/dashboard/project/_/integrations/queues/overview',
+    },
+    image: (
+      <Image
+        src={{
+          dark: '/images/modules/queues/queues-ui-dark.png',
+          light: '/images/modules/queues/queues-ui-light.png',
```
Comment (w3b6x9) [inline apps/www/data/products/modules/queues.tsx:239]: REMINDER: make sure these images are added.

File: apps/www/data/products/modules/queues.tsx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    cta: {
+      label: 'Start message queuing',
+      url: 'https://supabase.com/dashboard/project/_/integrations/queues/overview',
+    },
+    reverse: true,
+    image: (
+      <Image
+        src={{
+          dark: '/images/modules/queues/queues-api-dark.png',
+          light: '/images/modules/queues/queues-api-light.png',
```
Comment (w3b6x9) [inline apps/www/data/products/modules/queues.tsx:214]: REMINDER: make sure these images are added.

File: apps/docs/public/img/queues-quickstart-roles.png
Comment (w3b6x9) [inline apps/docs/public/img/queues-quickstart-roles.png:1]: REMINDER: swap out for updated permissions grid

File: apps/www/_blog/2024-12-05-supabase-queues.mdx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+---
+
+Today we're releasing [Supabase Queues](/modules/queues), for durable background task processing.
+
+Supabase Queues is a Postgres-native, durable Message Queue with guaranteed delivery, improving the scalability and resiliency of your applications. It's designed to work seamlessly with the entire Supabase platform.
+
+<div className="video-container mb-8">
+  <iframe
+    className="w-full"
+    src="https://www.youtube-nocookie.com/embed/miRQPbIJOuQ"
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-05-supabase-queues.mdx:25]: REMINDER: update video

File: apps/www/data/features.tsx [hunk 4]
```diff
@@ -2104,4 +2104,29 @@ OrioleDB is a PostgreSQL storage extension built on its pluggable storage framew
...
+## Features:
+1. Battle-tested Infrastructure: Built on PGMQ with proven production deployments and active maintenance.
+2. Native PostgreSQL Integration: Zero additional infrastructure, transactional consistency, and high performance.
+3. Row Level Security Integration: Native PostgreSQL RLS support for granular access control.
+4. Visual Queue Management: Built-in dashboard for queue monitoring and management.
+5. Message Archival: Built-in support for archiving processed messages for audit trails.
+`,
+    icon: Database,
+    products: [PRODUCT_SHORTNAMES.DATABASE, ADDITIONAL_PRODUCTS.PLATFORM],
+    heroImage: 'https://www.youtube-nocookie.com/embed/[INSERT_VIDEO_ID]',
```
Comment (w3b6x9) [inline apps/www/data/features.tsx:2124]: REMINDER: update video

File: apps/docs/public/img/queues-quickstart-roles.png
Comment (ivasilov) [reply apps/docs/public/img/queues-quickstart-roles.png:1]: Fixed.

**Review by fsansalvadore — APPROVED** [review]

**Review by saltcod — APPROVED** [review]

**Pull request merged**